pkgs/development/lua-modules/overrides.nix

```diff
@@ -350,18 +358,17 @@ with prev;
 
     buildInputs = [ pkgs.pkg-config pkgs.libuv ];
 
-    doInstallCheck = true;
```

Comment:
Why?

Reply:
because they've removed the tests from the tarball (NB: I had enabled the test in the previous update so I am a bit sad)